articles/quickstart/native/_includes/ionic/_configure_urls_interactive.md

```diff
@@ -10,9 +10,9 @@ Throughout this article, `YOUR_PACKAGE_ID` is your application's package ID. Thi
 
 ### Configure an application
 
-Use the interactive selector to create a new Auth0 application or select an existing application that represents the project with which you want to integrate. Every application in Auth0 is assigned an alphanumeric, unique client ID that your application code will use to call Auth0 APIs through the SDK.
+Use the interactive selector to create a new Auth0 application or select an existing application that represents the project you want to integrate with. Every application in Auth0 is assigned an alphanumeric, unique client ID that your application code will use to call Auth0 APIs through the SDK.
 
-Any settings you configure using this quickstart will automatically update for your application in the <a href="${manage_url}/#/">Dashboard</a>, which is where you can manage your Applications in the future.
+Any settings you configure using this quickstart will automatically update for your Application in the <a href="${manage_url}/#/">Dashboard</a>, which is where you can manage your Applications in the future.
```

Comment:
Is this change intentional? Seems like it should be `application` ?

Reply:
You're right, looks like I missed this when resolving the merge conflicts. Will fix